openapi/spec/path.py

```diff
@@ -24,6 +26,61 @@ def insert_data(self, data, *, strict=True, body_schema='body_schema'):
             data.update(path)
         return data
 
+    def get_order_clause(self, table, query, order_by, order_desc):
+        if not order_by:
+            return query
+
+        order_by_column = getattr(table.c, order_by)
+        if order_desc:
+            order_by_column = order_by_column.desc()
+        return query.order_by(order_by_column)
+
+    def limit_and_order_query(
+            self,
+            query,
+            table,
+            limit=DEF_PAGINATION_LIMIT,
+            offset=0,
+            order_by=None,
+            order_desc=False,
+    ):
+        if isinstance(query, Select):
+            query = self.get_order_clause(table, query, order_by, order_desc)
+            query = query.offset(offset)
+            query = query.limit(limit)
+        return query
```

Comment:
The above two methods, `get_order_clause` and `limit_and_order_query` don't belong in `ApiPath`. There's a `SqlApiPath` that's more appropriate, but let's minimise DB logic placed into these classes

Reply:
It also feels strange having these so far from the rest of the query generation